src/components/AppHeader.vue

```diff
@@ -53,35 +53,28 @@
       </div>
       <div class="nav-item">
         <CalendarIcon size="1x" aria-hidden="true" role="img" />
-        <VueCtkDateTimePicker
-          v-model="startDateTime"
-          :format="DATE_TIME_FORMAT"
-          :color="$config.primaryColor"
-          :locale="$config.locale"
-          :max-date="endDateTime"
-          :button-now-translation="$t('Now')"
+        <date-picker
+          v-model="rangeDateTime"
+          type="datetime"
+          format="YYYY-MM-DD HH:mm"
+          range
+          range-separator=" ~ "
```

Comment:
I think this should be a plain hyphen (`-`) or en dash (`–`)

Reply:
That dash makes it better to visually separate the dates so I went with it. I'll leave screenshots below